### DISCUSSION TITLE
Moar Activities

### DISCUSSION THREAD

**Discussion by littlesera** [littlesera/LSCG#162] [Ideas]
* [x] Nuzzle
  * [x] Nuzzle legs + feet
* [x] Bap
* [x] Grab Horn
* [x] Headbutt
* [x] Fellatio
* [x] Eat your own baguette
* [ ] 'Pour Drink' (can give drink to others with "sip," but this is maybe to use up all drink 'charges' at once for a larger effect)
  * [x] 'Pour into Funnel' specifically
* [x] Nibble Wings
* [x] "Tug Crotch Rope"
* [ ] ~"Attach Crotch Rope to Cuffs?" (not a lot that the script would do about this.. maybe best to leave to RP)~
* [x] Flick
* [x] Hug plush (shark and other animal plushes too)
* [ ] Steal panties
* [ ] Grab collar (essentially the same as FBC's leash-less leashing. Want to check and not step on toes here. Hand-gagging was moved to LSCG, so maybe this can too)
* [x] Chomp and hold? (bite + release, follows people out?)
* [x] Camera photography
* [ ] Dancing
* [ ] ~Penetrate with dildo gag [BC bug on male characters]~
* [ ] Persistent Hugs
* [ ] Hug-dragging

**Comment by dDeepLb** [comment]
grab collar or tug on collar would be nice too

**Comment by MaksimBondarev** [comment]
Pass the leash to another person

**Comment by ghost** [comment]
For the Foot, Pantyhose etc Lovers...
How About some Foot Actions? Like... "Nuzzles Foot/Nuzzles into Foot"

**Reply by AurelieCatena** [reply]
Tongue wash Foot

**Reply by littlesera** [reply]
Nuzzle implemented in #362

**Comment by AurelieCatena** [comment]
**Pour into funnel**

When someone has a funnel gag, she can't refuse to drink whatever is poured into it.

**Reply by littlesera** [reply]
Implemented in #385

**Comment by MaksimBondarev** [comment]
Breasts and butt smothering

**Comment by MaksimBondarev** [comment]
Give scissors ability to "cut" clothes
